src/PrettierLinter.php

```diff
@@ -42,11 +42,10 @@ public function getLinterConfigurationName() {
   }
 
   public function getDefaultBinary() {
-    list($err, $stdout, $stderr) = exec_manual('yarn -s --cwd %s which prettier', $this->getProjectRoot() . '/' . $this->cwd);
-    $binaryPath = strtok($stdout, "\n");
-    return $binaryPath;
+      return __DIR__ . "/.prettier-zero.sh";
```

Comment:
This will break the current implementation by ignoring the `--cwd` option. This requires that a `node_modules` be resolvable from the project root, which is why `--cwd` was added, to circumvent this.